backend/audit/etl.py

```diff
@@ -168,14 +169,15 @@ def load_revision(self):
         revision.save()
 
     def load_passthrough(self):
-        passthrough = Passthrough(
-            passthrough_name=None,  # TODO: Where does this come from?
-            passthrough_id=None,  # TODO: Where does this come from?
-            audit_id=-1,  # TODO: Where does this come from?
-            audit_year=self.audit_year,
-            report_id=self.report_id,
-        )
-        passthrough.save()
+        federal_awards = self.single_audit_checklist.federal_awards
+        for entry in federal_awards["FederalAwards"]["federal_awards"]:
+            passthrough = Passthrough(
+                award_seq_number=entry["award_reference"],
+                report_id=self.report_id,
+                passthrough_id=None,  # TODO: Where does this come from?
```

Comment:
They're coming from [`entities` ](https://github.com/GSA-TTS/FAC/pull/1491/files#diff-6805fa4d712052ec65a61ae143dac89fa623d90bb6ac7abbad5add80ee7f51feR24) array inside [`direct_or_indirect_award`](https://github.com/GSA-TTS/FAC/pull/1491/files#diff-6805fa4d712052ec65a61ae143dac89fa623d90bb6ac7abbad5add80ee7f51feR22)

Reply:
Added in new commit.